### ISSUE TITLE
graphql docker service does not respect --http-port 1337 --disable-tls true

### ISSUE THREAD

**Issue by andr-ec** [nhost/cli#754]
on a fresh install, if I run `nhost up --http-port 1337 --disable-tls true`
Then if I open the docker-compose.yaml, under the graphql service environment variables there is: 
```
            NHOST_AUTH_URL: https://local.auth.nhost.run/v1
            NHOST_BACKEND_URL: https://local.nhost.run
            NHOST_FUNCTIONS_URL: https://local.functions.nhost.run/v1
            NHOST_GRAPHQL_URL: https://local.graphql.nhost.run/v1
            NHOST_HASURA_URL: https://local.hasura.nhost.run
            NHOST_STORAGE_URL: https://local.storage.nhost.run/v1
```

Those urls should now be http and use the 1337 port.
This means that any request made from graphql to any of those (for example a remote schema with the stripe integration) will fail because they are going to the wrong port using the wrong protocol. 

I couldn't find any workarounds to make this work.

**Comment by dbarrosop** [comment]
You are right, this needds to be fixed in the CLI itself. Baseically [this](https://github.com/nhost/cli/blob/main/dockercompose/compose.go#L306-L311) needs to be added [here](https://github.com/nhost/cli/blob/main/dockercompose/graphql.go#L23-L26)

**Comment by andr-ec** [comment]
I was able to fix it in a fork: 
https://github.com/nhost/cli/compare/main...andr-ec:nhost-cli:tlsportfix

I can't really create a pull request since I made changes to the files in the vender folder that look like the pull from a private repo: nhost/be 
I also fixed the test that was checking that incorrectly.

Something else I noticed that should probably be fixed [here](https://github.com/nhost/cli/blob/e3dda411e6dc1ff24c5eb3ec4a1ea34ddaba8da8/dockercompose/compose.go#L443) is that it should be ports["hasura"]). ports["graphql"]) doesn't exist.
